tests/queries/0_stateless/01591_window_functions.sql

```diff
@@ -474,10 +474,10 @@ select count() over () from numbers(4) where number < 2;
 
 -- floating point RANGE frame
 select
-    count(*) over (order by toFloat32(number) range 5. preceding),
-    count(*) over (order by toFloat64(number) range 5. preceding),
-    count(*) over (order by toFloat32(number) range between current row and 5. following),
-    count(*) over (order by toFloat64(number) range between current row and 5. following)
+    count(*) over (order by toFloat32(number) range 5 preceding),
```

Comment:
@alexey-milovidov Allowing floating-point numbers as window frame bounds for `RANGE` is nonsense (for `ROWS` it's not allowed). Should we mark the change as backward-incompatible?

Reply:
Yes, let's make it backward incompatible, just in case.

But also, take the following arguments:
1. Other DBMS can support floating point numbers as bounds if they appear to be integers or even after rounding them.
2. Floating point numbers can appear if a query is generated by some generator, and the generator uses floating point as an intermediate representation for numbers, and used to format them this way.

Reply:
Well, at least currently it's inconsistent, - bounds for `ROWS`/`GROUPS` have to be 32-bit integers. If it's worth supporting floating-point numbers as bounds, can do it, but it should be consistent among all types of window frames.